### PR TITLE
Audio Player base finished.

### LINKED ISSUE #16: Adding seek function.
The audio player should be able to seek through the songs that are playing.

### PR DESCRIPTION
**Pull request by aon2003**
The Audio Player as of now has all the basic functions ready.
Code reformated, some bugs fixed.

closes #16

### CODE REVIEW

File: audio_player/midi.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            elif val == 'vol2':
+                self.volume_left.set(self.read_serial())
+            elif val == 'vol1':
+                self.volume_right.set(self.read_serial())
+            elif val == 'seek1':
+                self.song_left.set(self.read_serial())
+            elif val == 'seek2':
+                self.song_right.set(self.read_serial())
+            elif val == 'play1':
+                self.pl.play_song()
```
Comment (krispetrov) [inline audio_player/midi.py:53]: I think you should call all button functions with 0 as a parameter ?!

**Review by krispetrov — CHANGES_REQUESTED** [review]
Try to change all functions with self.pl.<function>(0)

File: audio_player/midi.py (hunk 1 above)
Comment (aon2003) [reply audio_player/midi.py:53]: Yes, that's true. Code will be fixed.

**Review by krispetrov — APPROVED** [review]
Now it it working all perfect and the communication is established

**Pull request merged**